pipeline/inputs/splunk.md

```diff
@@ -12,7 +12,9 @@ The **splunk** input plugin handles [Splunk HTTP HEC](https://docs.splunk.com/Do
 | buffer_max_size          | Specify the maximum buffer size in KB to receive a JSON message.                                                                              | 4M      |
 | buffer_chunk_size        | This sets the chunk size for incoming incoming JSON messages. These chunks are then stored/managed in the space available by buffer_max_size. | 512K    |
 | successful_response_code | It allows to set successful response code. `200`, `201` and `204` are supported.                                                              | 201     |
-| splunk\_token           | Add an Splunk token for HTTP HEC.`                                  |         |
+| splunk\_token           | Add an Splunk token for HTTP HEC. If specified multiple tokens with commas, it will be divided into each of tokens and they will be used for authentication of HTTP HEC. |         |
+| store\_token\_to\_metadata | Store Splunk HEC tokens to matadata. If set as false, they will be stored into records.                                                     | true    |
```

Comment:
```suggestion
| store\_token\_to\_metadata | Store Splunk HEC tokens as metadata. If set as false, they will be stored as records.                                                     | true    |
```

Reply:
Not sure what you mean here by metadata - is that on the Splunk side so nothing to do with FB metadata/records?

Reply:
It's a fluent-bit side of metadata:

```
[[[timestamp, {metadata}], {record objects}]]
```

ref: https://github.com/fluent/fluentd/wiki/Forward-Protocol-Specification-v1.5#forward-mode